sdk/python/kfp/dsl/__init__.py

```diff
@@ -24,4 +24,7 @@
 from ._volume_snapshot_op import VolumeSnapshotOp
 from ._ops_group import OpsGroup, ExitHandler, Condition
 from ._component import python_component, graph_component, component
-from ._artifact_location import ArtifactLocation
\ No newline at end of file
+from ._artifact_location import ArtifactLocation
+
+task_id_placeholder = '{{workflow.uid}}-{{pod.name}}'
```

Comment:
you might as well add all the argo dynamic params in the dsl and create a submodule called e.g. dsl.param

Reply:
The other ones seems to already been hidden by DSL (e.g. `{{inputs.*}}`, `{{tasks.*}}` etc).
What placeholders do you have in mind?

P.S. I want the DSL placeholders to be portable, so that they can be supported outside Argo.

Reply:
can we just use {{workflow.uid}} as discussed offline? and backend can replace with the run id.

Reply:
> can we just use {{workflow.uid}} as discussed offline? and backend can replace with the run id.

I do remember our discussion. Do you think something needs to be changed in this PR?

Reply:
Took a second look. The change LGTM